Web/starter_project_vanguard/tailwind.config.js

```diff
@@ -14,12 +14,19 @@ module.exports = {
         danger: '#b91c1c',
         'primary-text': '#353535',
         'secondary-text': '#717171',
+        'tertiary-text': '#565656',
+        
       },
       backgroundImage: {
         'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
+      fontFamily: {
```

Comment:
nice
I have seen someone else that did this as well so if there exists conflict when merging just be careful
otherwise I like it

Reply:
Okay.